bindings/Sofa/package/__init__.py

```diff
@@ -364,3 +364,15 @@ def doReInit(self):
     SofaPrefabF.__dict__["__original__"] = f
     return SofaPrefabF
 
+def import_sofa_python_scene(path_to_scene : str):
+    """Return a python module containing a sofa scene"""
+    spec = importlib.util.spec_from_file_location("sofa.scene", path_to_scene)
+    foo = importlib.util.module_from_spec(spec)
```

Comment:
A better name perhaps?

Reply:
any suggestion ?

Reply:
It is a python function so it should probably be for an example.